src/translations/nbTranslations.json

```
@@ -18,10 +18,10 @@
     "short_description": "Nasjonalt vitenarkiv (NVA) samler informasjon om norsk forskning og gjør forskningsresultater og forskningsinformasjon åpent tilgjengelig."
   },
   "authorization": {
-    "about_terms": "Om brukervilkår",
-    "about_terms_description": "<0>For å bruke tjenesten Nasjonalt vitenarkiv må du akseptere <0>Sikts brukervilkår<0></0></0> for Nasjonalt vitenarkiv.</0><0>Om du legger inn personopplysninger om andre personer må du ha deres tillatelse eller annet lovlig grunnlag. Du må sikre at vedkommende er informert om at personopplysningene publiseres i Nasjonalt vitenarkiv.</0><0>Du kan ikke tilgjengeliggjøre/laste opp opphavsrettslig materiale/åndsverk uten å ha tillatelse fra alle opphavere.</0><0>Du er ansvarlig for at din bruk av Nasjonalt vitenarkiv ikke er i strid med gjeldende rett.</0><0>For mer informasjon, se <0>Sikts brukervilkår<0></0></0> i sin helhet.</0><1>Ved å trykke \"$t(authorization.accept)\" aksepterer du Sikt sine brukervilkår.</1>",
+    "about_terms": "Brukervilkår og personvernerklæring for Nasjonalt vitenarkiv",
+    "about_terms_description": "<p>For å bruke tjenesten må du akseptere Sikts <a1>Brukervilkår for Nasjonalt vitenarkiv</a1>, og gjøre deg kjent med <a2>Personvernerklæring for Nasjonalt vitenarkiv.</a2></p><p>Ved å klikke på «Godta», bekrefter du at du har lest og akseptert brukervilkårene. Du samtykker også til behandling av personopplysninger i samsvar med personvernerklæringen.</p>",
```

D
Noen grunn til å ta vekk bruken av annen nøkkel for Godta-knappen?
Vanskelig å vite hva relatert knapp er når man ser dette i Lokalise, så blir fort forveksling mellom godta/fullfør/lagre/etc for ulike språk

```suggestion
    "about_terms_description": "<p>For å bruke tjenesten må du akseptere Sikts <a1>Brukervilkår for Nasjonalt vitenarkiv</a1>, og gjøre deg kjent med <a2>Personvernerklæring for Nasjonalt vitenarkiv.</a2></p><p>Ved å klikke på \"$t(authorization.accept)\", bekrefter du at du har lest og akseptert brukervilkårene. Du samtykker også til behandling av personopplysninger i samsvar med personvernerklæringen.</p>",
```